Detectors/TOF/base/include/TOFBase/CalibTOFapi.h

```diff
@@ -107,17 +117,17 @@ class CalibTOFapi
   Diagnostic* mDiaFreq = nullptr;  ///< object for Diagnostic Frequency
 
   // info from diagnostic
-  int mNoisyThreshold = 1;                          ///< threshold to be noisy
-  float mEmptyTOF = 0;                              ///< probability to have TOF fully empty
-  float mEmptyCrateProb[Geo::kNCrate];              ///< probability to have an empty crate in the current readout window
-  std::vector<std::pair<int, float>> mNoisy;        ///< probTRMerror
-  std::vector<std::pair<int, float>> mTRMerrorProb; ///< probTRMerror
-  std::vector<int> mTRMmask;                        ///< mask error for TRM
-
-  bool mIsErrorCh[Geo::NCHANNELS];  ///< channels in error (TRM)
-  std::vector<int> mFillErrChannel; ///< last error channels filled
-  bool mIsOffCh[Geo::NCHANNELS];    ///< channels in error (TRM)
-  bool mIsNoisy[Geo::NCHANNELS];    ///< noisy channels
+  int mNoisyThreshold = 1;                           ///< threshold to be noisy
+  float mEmptyTOF = 0;                               ///< probability to have TOF fully empty
+  float mEmptyCrateProb[Geo::kNCrate] = {INITF_ALL}; ///< probability to have an empty crate in the current readout window
```

Comment:
you don't need these `{XXX}` with defines, just use `{}`.

Reply:
Sorry, I didn't understand the comment. What are you proposing?

Reply:
I mean remove all these `#define`s and simple declare your datamembers as
```
float mEmptyCrateProb[Geo::kNCrate] = {};
...
bool mIsErrorCh[Geo::NCHANNELS] = {};
bool mIsOffCh[Geo::NCHANNELS] = {};
bool mIsNoisy[Geo::NCHANNELS] = {};
```

Reply:
done